### ISSUE TITLE
Handle error return from omni on create sliver

### ISSUE THREAD

**Issue by tcmitchell** [GENI-NSF/geni-portal#321]
When omni returns an error on create sliver, instead of the manifest rspec, the portal shows:
```
Reserved resources on AM (InstaGENI Utah) until 2012-11-06 05:43:02 UTC:
Rspec returned was not valid XML.
```
This is incorrect, resources were not reserved. And the error message is logged but not shown to the user, so they don't know what went wrong (or even that something went wrong), and don't have any hope of fixing it.

Improve the error handling by indicating that resources were nto reserved and showing the error message.

 _Imported from trac ticket [#321](http://trac.gpolab.bbn.com/proto-ch/ticket/321),  created by tmitchel on 11-06-2012 at 10:24, last modified: 07-10-2013 at 01:24_

**Comment by seledwards** [comment]
To test: Try sending a malformed request RSpec.

_Trac comment by sedwards (github user: seledwards) on 02-12-2013 at 15:51_

**Comment by ahelsing** [comment]
This appears to have been fixed by #649. As an example, when a malformed XML file is sent to GPO InstaGENI from cascade, the following is now shown to the user:

```
Error: Failed to create a sliver. Check log file at https://boss.instageni.gpolab.bbn.com/spewlogfile.php3?logfile=2182c2773cebd2f57e4d4905156fe6d1

Failed CreateSliver for slice urn:publicid:IDN+ch1.gpolab.bbn.com:BLAH+slice+aslice2 at https://www.instageni.gpolab.bbn.com:12369/protogeni/xmlrpc/am/2.0. Error from Aggregate: code 2. protogeni AM code: 2: rspec is not well formed (PG log url - look here for details on any failures: https://boss.instageni.gpolab.bbn.com/spewlogfile.php3?logfile=2182c2773cebd2f57e4d4905156fe6d1).
```

_Trac comment by bujcich on 07-05-2013 at 09:36_
